test/utils.test.js

```diff
@@ -28,8 +29,10 @@ describe('isInput', () => {
   });
 });
 
+let window;
 describe('matchesKeyboardEvent', () => {
   describe('matches KeyboardEvent.key against', () => {
+    window = (new JSDOM('', { runScripts: 'dangerously' })).window;
```

Comment:
can you do `const window` here?

Reply:
Actually I had a quick look at this and the JSDOM wasn't required for any of the tests. I guess the setup initialized everything in the end. This was me trying to follow this tip: https://github.com/jsdom/jsdom/wiki/Don%27t-stuff-jsdom-globals-onto-the-Node-global

If I remember it correctly then removing the setup will cause issues for the `mount` in the components. We could though make all the tests use a local window here so that separation is improved.

Reply:
Just tested with a helper for getting the event:

```js
const getKeyboardEvent = (eventName: string, opts: {| key?: string, keyCode?: number |}) => {
  const { window: testWindow } = (new JSDOM('', { runScripts: 'dangerously' }));
  return new testWindow.KeyboardEvent(eventName, opts);
};
```

Works nicely and should keep the global window unaffected.